test/functional/mempool_compatibility.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that mempool.dat is compatible between versions
+
+Download node binaries:
+contrib/devtools/previous_release.sh -b v0.19.1
```

Comment:
nit: I prefer to have the same instruction in all test files that require previous releases, because otherwise other tests fail when they have some versions but not all. Try:
```
contrib/devtools/previous_release.sh -b v0.19.1 v0.18.1 v0.17.1 v0.16.3 v0.15.2

Only v0.19.1 is required by this test. The rest is used in other backwards compatibility tests.
```